test/prima_auth0_ex/token_provider/token_encryptor_test.exs

```diff
@@ -0,0 +1,57 @@
+defmodule PrimaAuth0Ex.TokenProvider.TokenEncryptorTest do
+  @moduledoc false
+
+  use ExUnit.Case, async: false
+
+  alias PrimaAuth0Ex.TestHelper
+  alias PrimaAuth0Ex.TokenProvider.TokenEncryptor
+
+  @token_encryption_key "eT4YutFXY/PCV5Kr6gBD/K2NxM60OqeXFux09te6Z80="
+
+  test "encrypts and decrypts correctly" do
+    TestHelper.set_client_env(:cache_encryption_key, @token_encryption_key, true)
+
+    plaintext = "test"
+
+    {:ok, enc} = TokenEncryptor.encrypt(plaintext)
+    {:ok, dec} = TokenEncryptor.decrypt(enc)
+
+    assert plaintext == dec
+  end
+
+  test "decrypting returns with :error if key changes" do
+    TestHelper.set_client_env(:cache_encryption_key, @token_encryption_key, true)
+
+    plaintext = "test"
```

Comment:
Can this variable be inlined in this specific test? The value isn't used afterwards.